docs/source/getting_started/distributions/self_hosted_distro/together.md

```diff
@@ -1,62 +1,67 @@
-# Together Distribution
-
-### Connect to a Llama Stack Together Endpoint
-- You may connect to a hosted endpoint `https://llama-stack.together.ai`, serving a Llama Stack distribution
+# Fireworks Distribution
```

Comment:
Wondering which sections gets auto-generated, and should we update the commands to build documentation in [CONTRIBUTING.md](https://github.com/meta-llama/llama-stack/blob/main/CONTRIBUTING.md#building-the-documentation)?

Reply:
I added a section to CONTRIBUTING.md but we also need a codegen pre-commit check to ensure there isn't a diff